ecs_dashboard/ecs_dashboard.airplane.tsx

```diff
@@ -0,0 +1,133 @@
+import {
+  Button,
+  DescriptionList,
+  Divider,
+  Heading,
+  Stack,
+  Table,
+  Text,
+  useComponentState,
+  showNotification,
+} from "@airplane/views";
+import airplane from "airplane";
+
+const EcsDashboard = () => {
```

Comment:
```suggestion
const ECSDashboard = () => {
```

(style nit, abbrevations should be uppercased)

Reply:
Noted, thanks!